### ISSUE TITLE
Review: Small Bugs Meta Issue

### ISSUE THREAD

**Issue by wxtim** [cylc/cylc-flow#4261]
Meta issue, describing current known problems with Cylc Review in the 7.8.x branch, esp, but not exclusively, WRT Cylc 8 Workflows.

Some of these issues have their own issues open in GH, and in this case I have listed rather than describing the issue.

## Bugs
- [x] Log files with timezone (#4260 #4276 ). 
- [ ] Failed jobs from Cylc 8 workflows not showing (#4207 ).
- [x] Running Cylc 8 Workflows showing as stopped under some circumstances. (#4280 )
- [x] Syntax of `cylc.flow` files is not highlighted (#4264 ). 

**Pull requests welcome!**
This is an Open Source project - please consider contributing a bug fix
yourself (please read `CONTRIBUTING.md` before starting any work though).

**Comment by wxtim** [comment]
All the otherwise undocumented issues referenced here are closed.
The unclosed one can't be reproduced, and has its own ticket anyway.
